### PR TITLE
align prod stable and prod-beta envs with stage

### PR DESCRIPTION
**Pull request by mkholjuraev**
This aligns the prod beta and stable envs with the stage env for RHEL navigation bundles. 

1. Only difference is Learning Resouces nav is not included per PMs ask
2. I did change only RHEL-related settings in the services.json file and this may need a closer review.

Putting this as Draft until we get an ack for release.

### CODE REVIEW

File: static/beta/prod/navigation/rhel-navigation.json [hunk 1]
```diff
@@ -1,318 +1,369 @@
...
+          "description": "Build and manage Red Hat Enterprise Linux images and environments."
+        },
+        {
+          "title": "System Configuration",
+          "expandable": true,
+          "routes": [
+            {
+              "id": "rhc",
+              "appId": "connector",
+              "title": "Remote Host Configuration (RHC)",
```
Comment (johnsonm325) [inline static/beta/prod/navigation/rhel-navigation.json:55]: should this be lowercase `(rhc)`?

File: static/beta/prod/navigation/rhel-navigation.json [hunk 2]
```diff
@@ -1,318 +1,369 @@
...
+              "href": "/settings/connector",
+              "product": "Red Hat Insights",
+              "description": "Configure your systems to execute Ansible Playbooks."
+            },
+            {
+              "id": "activationKeys",
+              "appId": "connector",
+              "title": "Activation Keys",
```
Comment (johnsonm325) [inline static/beta/prod/navigation/rhel-navigation.json:63]: Should be `Activation keys`.

File: static/beta/prod/navigation/rhel-navigation.json [hunk 3]
```diff
@@ -1,318 +1,369 @@
...
+              "product": "Red Hat Insights"
+            }
+          ]
+        },
+        {
+          "appId": "policies",
+          "title": "Policies",
+          "href": "/insights/policies"
+        }
+      ]
```
Comment (johnsonm325) [inline static/beta/prod/navigation/rhel-navigation.json:172]: mocks show `Resource optimization` here.

File: static/beta/prod/navigation/rhel-navigation.json [hunk 4]
```diff
@@ -1,318 +1,369 @@
...
+          "title": "Subscriptions",
+          "routes": [
+            {
+              "id": "rhel",
+              "appId": "subscriptions",
+              "title": "RHEL",
+              "href": "/insights/subscriptions/rhel",
+              "product": "Subscription Watch"
+            },
+            {
```
Comment (johnsonm325) [inline static/beta/prod/navigation/rhel-navigation.json:272]: Mocks show `Satellite` after `RHEL`.

File: static/beta/prod/navigation/rhel-navigation.json [hunk 5]
```diff
@@ -1,318 +1,369 @@
...
+          "isExternal": true
+        },
+        {
+          "id": "apis",
+          "appId": "apiDocs",
+          "title": "APIs",
+          "href": "/docs/api"
+        }
+      ]
+    }
```
Comment (johnsonm325) [inline static/beta/prod/navigation/rhel-navigation.json:367]: I'm not seeing any of these in the mocks. Perhaps I'm looking at the wrong mocks?

Comment (mkholjuraev) [reply static/beta/prod/navigation/rhel-navigation.json:367]: Yup, this does not exist in the mockups. this is what PMs wanted

File: static/beta/prod/navigation/rhel-navigation.json (hunk 1 above)
Comment (mkholjuraev) [reply static/beta/prod/navigation/rhel-navigation.json:55]: It is in capital in the [mocks](https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/a/mPowdv7)

File: static/beta/prod/navigation/rhel-navigation.json (hunk 3 above)
Comment (mkholjuraev) [reply static/beta/prod/navigation/rhel-navigation.json:172]: This latest [mock](https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/a/mPowdv7) shows it under Business group

File: static/beta/prod/navigation/rhel-navigation.json (hunk 2 above)
Comment (mkholjuraev) [reply static/beta/prod/navigation/rhel-navigation.json:63]: Nice catch! Will fix this

File: static/beta/prod/navigation/rhel-navigation.json (hunk 4 above)
Comment (mkholjuraev) [reply static/beta/prod/navigation/rhel-navigation.json:272]: Added

File: static/beta/prod/navigation/rhel-navigation.json (hunk 2 above)
Comment (mkholjuraev) [reply static/beta/prod/navigation/rhel-navigation.json]: Fixed

**Comment by mkholjuraev** [comment]
@Hyperkid123 I got the green light to update the prod env also with new bundle changes. Please have a look at this and let me know if there is anything that needs to be fixed/changed.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**